app/models/submission.rb

```diff
@@ -41,11 +41,17 @@ def exercise_version
   end
 
   def serialized
+    tests_data = tests_status
+    if tests_exceptioned?
+      job = ToolingJob.find(test_runs.last.tooling_job_id, full: true)
+      tests_data += "\n\n#{JSON.pretty_generate(job.execution_metadata)}"
+    end
```

Comment:
This is temporary so I didn't have to fiddle with the React, but it's fine for now.